cmd/openshift-install/agent.go

```diff
@@ -129,6 +133,9 @@ func newAgentCreateCmd(ctx context.Context) *cobra.Command {
 	for _, t := range agentTargets {
 		t.command.Args = cobra.ExactArgs(0)
 		t.command.Run = runTargetCmd(ctx, t.assets...)
+		if t.name == "Agent ISO Image" {
+			t.command.PersistentFlags().BoolVar(&agentImageMinimalISO, "minimal-iso", false, "generates a minimal ISO image, by default a full ISO will be generated.")
```

Comment:
I'm inclined to think this should be a flag in the agent-config instead of a CLI option. The reason to use a minimal ISO is that some hardware requires it, so that config should travel along with the hardware configuration, rather than being something the user has to remember to do at runtime.

I originally imagined we would trigger this off the presence of the `bootArtifactsBaseURL`, but I guess there is a use case for a connected cluster using a minimal ISO where we want to just use the mirror URL.

Reply:
That seems a fair point to me (let the config travel along witht hw config) for this specific case, it could be worth to be explored

Reply:
Reworked to use MinimalISO configuration from agent-config.yaml instead of command line flag.